### PR TITLE
Volt Void is now a more useful scripture

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
experiment: Redesigned: Volt Void now allows you to fire up to 5 energy rays at targets in view; each ray does 25 laser damage to non-Servants in the target tile. The ray will consume power to do up to double damage, however.
wip: Failing to fire a Volt Void ray will damage you, though you won't die from it unless you have access to a lot of power and are either low on health or fail all five rays in a row.
/:cl:

Three bugs fixed here:
1. A few scriptures didn't actually set themselves as successful on doing stuff.
2. Even for scriptures that DID, it'd exit out of the loop that set if it was successful before managing to set it, meaning none of them were considered successful ever.
3. run_scripture() always returned TRUE, even for failed scripture recital, meaning anything relying on it returning FALSE for anything would be very disappointed.

### CODE REVIEW

**Comment by Shadowlight213** [comment]
does standing on a transmission sigil make you take damage if you cant fire a ray?

**Comment by ChangelingRain** [comment]
Yes*
*you can always fire a ray, and getting stunned/moved/etc only interrupts the scripture, it doesn't make you take ray damage

**Comment by Shadowlight213** [comment]
So, this means that charging sigils will damage you now?

Well thats shit.

**Comment by ChangelingRain** [comment]
It doesn't charge sigils anymore; the interdiction lens is used for that
Why is this a combat scripture now? Because volt void was pretty much fucking useless at the tier it's at, and going up one tier gets you interdiction lenses, which are Just Better.

**Comment by GunHog** [comment]
**You need to remove the 2 minute shutdown on Interdiction Lens. That structure is terrible and has too many rules and reqs!**

**Comment by ChangelingRain** [comment]
If the interdiction lens is shutting down you've put it probably literally in space next to nothing it can send power to.
IE; my answer is no, it's not hard to use, the rules are "is there an APC in the area it's in or is there a sigil next to it"

File: code/game/gamemodes/clock_cult/clock_scriptures/scripture_scripts.dm [hunk 1]
```diff
@@ -268,62 +268,64 @@
 	return slab.procure_gateway(invoker, duration, portal_uses)
 
 
-//Volt Void: Channeled for up to thirty times over thirty seconds. Consumes power from most power storages and deals slight burn damage to the invoker.
+//Volt Void: Channeled for up to five times over ten seconds to fire up to five rays of energy at target locations.
 /datum/clockwork_scripture/channeled/volt_void
-	descname = "Channeled, Area Power Drain"
+	descname = "Channeled, Targetted Energy Blasts"
```
Comment (AdamElTablawy) [inline code/game/gamemodes/clock_cult/clock_scriptures/scripture_scripts.dm]: targeted, not targetted

File: code/game/gamemodes/clock_cult/clock_scripture.dm [hunk 2]
```diff
@@ -263,6 +267,7 @@ Judgement: 12 servants, 5 caches, 300 CV, and any existing AIs are converted or
 	var/ranged_type = /obj/effect/proc_holder/slab
 	var/ranged_message = "This is a huge goddamn bug, how'd you cast this?"
 	var/timeout_time = 0
+	var/allow_mobility = TRUE //if moving and swapping hands is allowed during the while
```
Comment (AdamElTablawy) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: Why does this default to true?

Comment (ChangelingRain) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: Because all previous scriptures that used this allowed moving during the channel

**Comment by KorPhaeron** [comment]
Frozen

**Pull request merged**